styles.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "inter", sans-serif;
+  color: white;
+}
+
+div {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+nav {
+  height: 10vh;
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  width: auto;
+  margin: 0;
+}
+
+.logo-text {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  padding-left: 24px;
+}
+
+.menu {
+  width: 40px;
+  height: 40px;
+  display: block;
+  align-self: center;
+  align-content: center;
+  padding: 10px 10px;
+  float: right;
+  margin-right: 20px;
+}
+
+#check {
+  display: none;
+}
+
+.main-line {
+  margin: 6px 24px;
+  color: #ff5b00;
+  font-size: 56px;
+  line-height: 64px;
+  font-family: 'Crete Round', sans-serif;
+}
```

Comment:
- Can we kindly stick to the font size on Figma as much as possible? The content of your page's main title currently looks bigger than what's on Figma and this is because of the large font size. Can we fix this? You should consider reducing the `font-size: 56px;` and font-weight too.

Here is what your page looks like 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="384" alt="Screenshot 2023-01-12 at 18 32 17" src="https://user-images.githubusercontent.com/99246359/212209997-ba60103b-4943-42e4-87c2-559653431bea.png"> |<img width="391" alt="Screenshot 2023-01-12 at 18 33 23" src="https://user-images.githubusercontent.com/99246359/212210054-ba430b24-a4b8-44c5-b206-6abf456352ba.png">  |


⚠️ Kindly take note of the text overlapping on the background image in your design. 🌸

Reply:
@Favourezeugwa The font size and font weight are already the same but they still differ in size.